Integrations/integration-Zscaler.yml

```diff
@@ -0,0 +1,118 @@
+description: Zscaler is a cloud security solution built for performance and flexible
+  scalability.
+configuration:
+- display: Cloud Name (i.e. admin.zscalertwo.net)
+  name: cloud
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Credentials
+  name: credentials
+  defaultvalue: ""
+  type: 9
+  required: true
+- display: API Key
+  name: key
+  defaultvalue: ""
+  type: 4
+  required: true
+- display: Do not validate server certificate (insecure)
+  name: insecure
+  defaultvalue: "false"
+  type: 8
+  required: false
+script:
+  script: |
+    ''' IMPORTS '''
+    import requests
+    import time
+    # disable insecure warnings
+    requests.packages.urllib3.disable_warnings()
+
+    ''' GLOBAL VARS '''
+    CLOUD_NAME=demisto.params()['cloud']
+    USERNAME=demisto.params()['credentials']['identifier']
+    PASSWORD=demisto.params()['credentials']['password']
+    API_KEY=str(demisto.params()['port'])
+    BASE_URL=CLOUD_NAME + '/api/v1'
+    USE_SSL = not demisto.params().get('insecure', False)
+    DEFAULT_HEADERS = {
+        'content-Type': 'application/json'
+    }
+
+    ''' HELPER FUNCTIONS '''
+    def http_request(method, url_suffix, data, headers):
+        data = {} if data is None else data
+        LOG('running request with url=%s\tdata=%s\theaders=%s' % (BASE_URL + url_suffix,
+            data, headers))
+        try:
+            res = requests.request(method,
+                BASE_URL + url_suffix,
+                verify=USE_SSL,
+                data=data,
+                headers=headers
+            )
+            if res.status_code not in (200, 204):
+                raise Exception('Your request failed with the following error: ' + res.reason)
+        except Exception, e:
+            LOG(e)
+            raise
+        return res.json()
+
+    ''' FUNCTIONS '''
+
+    def login():
+        cmd_url = '/authenticatedSession'
+        data = {
+            'username': USERNAME,
+            'password': PASSWORD,
+            'apiKey': API_KEY,
+            'timestamp': int(round(time.time() * 1000))
+        }
+        res = http_request('POST', cmd_url, data, DEFAULT_HEADERS)
+
+    def logout():
+        cmd_url = '/authenticatedSession'
+        res = http_request('DELETE', cmd_url, None, DEFAULT_HEADERS)
+
+    def blacklist_url(url):
+        cmd_url = '/security/advanced/blacklistUrls?action=ADD_TO_LIST'
+        blacklistUrls = url.split(',')
+        data = {
+            'blacklistUrls': blacklistUrls
+        }
+        res = http_request('POST', cmd_url, data, DEFAULT_HEADERS)
+        return 'Added to blacklist successfully.'
```

Comment:
might be good to MD the list to the war room (easier to read than the command itself)